### PR TITLE
Test/reviews

### PR DESCRIPTION
**Pull request by oskber**
- Cypress test for posting reviews.
- Some refactoring of ReviewsList component
- A bunch of auto formatting on other files

image

### CODE REVIEW

**Review by ANschoolacc — APPROVED** [review]
We really should implement github action to auto format on push or a hook that formats on commit, but good that we now have everything formatted as it should be. Your descriptions in your tests makes them easy to understand. Nice job!

File: kino-nextjs/app/components/reviews/ReviewsList.tsx [hunk 1]
```diff
@@ -8,37 +8,40 @@ import Link from 'next/link';
...
+  ) {
     if (i >= 1 && i <= totalPages) {
       pageNumbers.push(i);
     }
   }
 
   return (
     <>
-      <div className='flex flex-col rounded'>
+      <div data-cy={'review-list'} className='flex flex-col rounded'>
```
Comment (MickeMarten) [inline kino-nextjs/app/components/reviews/ReviewsList.tsx:44]: Is data.cy insetead of classnames?

File: kino-nextjs/app/components/reviews/ReviewsList.tsx [hunk 2]
```diff
@@ -67,7 +70,7 @@ export default async function ReviewsList({
         ) : (
           <div className='flex justify-center items-center mt-7'>
             <div className='flex border-[1px] gap-4 rounded border-custom_yellow p-2'>
-              {page === 1 ? (
+              {currentPage === 1 ? (
```
Comment (MickeMarten) [inline kino-nextjs/app/components/reviews/ReviewsList.tsx:73]: Great change, more understandable.

File: kino-nextjs/cypress.config.ts
Comment (MickeMarten) [inline kino-nextjs/cypress.config.ts:1]: So after cypress makes a comment it also removes the comment from the database?
Very inventive, good move!

**Review by MickeMarten — APPROVED** [review]
The tests are easy to read and easy to understand, great job implementing it.

File: kino-nextjs/app/components/reviews/ReviewsList.tsx (hunk 1 above)
Comment (oskber) [reply kino-nextjs/app/components/reviews/ReviewsList.tsx:44]: > Is data.cy insetead of classnames?

Exactly! The data-cy attribute isolates them from other selectors and makes it easier to target elements :)

**Comment by oskber** [comment]
> We really should implement github action to auto format on push or a hook that formats on commit, but good that we now have everything formatted as it should be. Your descriptions in your tests makes them easy to understand. Nice job!

I agree that we should add actions or hooks!

**Pull request merged**